src/views/poi_bloc/Images.jsx

```diff
@@ -0,0 +1,55 @@
+/* global _ */
+import React from 'react';
+import classnames from 'classnames';
+
+function createImage(param, classes) {
+  return <a
+    target='_blank'
+    rel='noopener noreferrer'
+    href={ param.href }
+    className={ classes }
+    style={{ backgroundImage: param.url }}
+  />;
+}
+
+function createImages(images) {
```

Comment:
I know this is a direct transcription of the current code but I'm sure the flow logic can be improved and simplified to just look like a loop over the image array, not having to test all these special cases of 1, 2, etc. But it probably requires some CSS changes, so maybe we can merge like that and improve later.
What do you prefer?

Reply:
If you have an idea on how to make it more straightforward, I'm totally fine into reworking it. I just don't know how to make it better. :-/

Reply:
The general idea would be to always generate the images with an array loop, not caring about their number, and implementing the different cases in CSS only because I think it's possible. Or at least it's possible with just a specific class on the container.
But I don't know if you like playing with CSS, that's why I asked for your preference. I can do it later if you want :)

Reply:
I'm not sure to fully understand what you have in mind so I'd prefer to let you handle it if you don't mind.